diff -Nur themes/ShinyMetal.orig/coords/coords.cfg themes/ShinyMetal/coords/coords.cfg
--- themes/ShinyMetal.orig/coords/coords.cfg	1999-10-19 05:24:30.000000000 +0200
+++ themes/ShinyMetal/coords/coords.cfg	2000-12-28 19:14:05.000000000 +0100
@@ -1,3 +1,13 @@
+/*
+Jan 27 2000, Japanese Localization by Masahiko Mori <masa@engr.sgi.com>
+                      Traslation by Ryuichi Wakita <wakita@xtrans.com>
+
+このファイルの日本語訳の部分、その他の変更部分は新規に作成したので上記２人に
+著作権があり、GPL に従って自由に配布できます。それ以外の変更のない部分につい
+ては原作の制限に従うものとします。
+このファイルに変更を加えた場合、このコメントに続き簡単な履歴を追加してください。
+*/
+
 __ICLASS __BGN
   __NAME COORDS
   __NORMAL "images/dock1.png"
@@ -8,7 +18,7 @@
 
 __TCLASS __BGN
   __NAME "COORDS"
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/ShinyMetal.orig/dialogs/dialogs.cfg themes/ShinyMetal/dialogs/dialogs.cfg
--- themes/ShinyMetal.orig/dialogs/dialogs.cfg	1999-10-19 05:24:31.000000000 +0200
+++ themes/ShinyMetal/dialogs/dialogs.cfg	2000-12-28 19:14:33.000000000 +0100
@@ -1,3 +1,13 @@
+/*
+Jan 27 2000, Japanese Localization by Masahiko Mori <masa@engr.sgi.com>
+                      Traslation by Ryuichi Wakita <wakita@xtrans.com>
+
+このファイルの日本語訳の部分、その他の変更部分は新規に作成したので上記２人に
+著作権があり、GPL に従って自由に配布できます。それ以外の変更のない部分につい
+ては原作の制限に従うものとします。
+このファイルに変更を加えた場合、このコメントに続き簡単な履歴を追加してください。
+*/
+
 __ICLASS __BGN
   __NAME DIALOG
   __NORMAL "dialogs/images/bg.png"
@@ -134,7 +144,7 @@
   __NAME "DIALOG"
   __JUSTIFICATION 512
 		
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -145,7 +155,7 @@
   __NAME "DIALOG_WIDGET_BUTTON"
   __JUSTIFICATION 512
   
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -156,7 +166,7 @@
   __NAME "DIALOG_WIDGET_CHECK_BUTTON"
   __JUSTIFICATION 0
   
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -167,7 +177,7 @@
   __NAME "DIALOG_WIDGET_RADIO_BUTTON"
   __JUSTIFICATION 0
   
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -178,7 +188,7 @@
   __NAME "DIALOG_WIDGET_TEXT"
   __JUSTIFICATION 512
   
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -189,7 +199,7 @@
   __NAME "DIALOG_BUTTON"
   __JUSTIFICATION 512
     
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/ShinyMetal.orig/epplets/epplets.cfg themes/ShinyMetal/epplets/epplets.cfg
--- themes/ShinyMetal.orig/epplets/epplets.cfg	1999-10-19 05:24:31.000000000 +0200
+++ themes/ShinyMetal/epplets/epplets.cfg	2000-12-28 19:16:16.000000000 +0100
@@ -1,3 +1,13 @@
+/*
+Jan 27 2000, Japanese Localization by Masahiko Mori <masa@engr.sgi.com>
+                      Traslation by Ryuichi Wakita <wakita@xtrans.com>
+
+このファイルの日本語訳の部分、その他の変更部分は新規に作成したので上記２人に
+著作権があり、GPL に従って自由に配布できます。それ以外の変更のない部分につい
+ては原作の制限に従うものとします。
+このファイルに変更を加えた場合、このコメントに続き簡単な履歴を追加してください。
+*/
+
 __ICLASS __BGN
   __NAME EPPLET_BACKGROUND_HORIZONTAL
   __NORMAL "epplets/images/bg_h.png"
@@ -286,7 +296,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_LABEL"
-  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--10-*,-*-r-*--10-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -295,7 +305,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_BUTTON"
-  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--10-*,-*-r-*--10-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -304,7 +314,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TOGGLEBUTTON_ON"
-  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--10-*,-*-r-*--10-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -313,7 +323,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TOGGLEBUTTON_OFF"
-  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--10-*,-*-r-*--10-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -322,7 +332,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_POPUP"
-  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-8-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--10-*,-*-r-*--10-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -331,7 +341,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TEXT_TINY"
-  __NORMAL "-*-lucida-medium-r-normal-*-*-80-*-*-p-*-iso8859-1"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-80-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--10-*,-*-r-*--10-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -339,7 +349,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TEXT_MEDIUM"
-  __NORMAL "-*-lucida-medium-r-normal-*-*-100-*-*-p-*-iso8859-1"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-100-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -347,7 +357,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TEXT_LARGE"
-  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--16-*,-*-r-*--16-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/ShinyMetal.orig/focuslist/focuslist.cfg themes/ShinyMetal/focuslist/focuslist.cfg
--- themes/ShinyMetal.orig/focuslist/focuslist.cfg	1999-10-19 05:24:31.000000000 +0200
+++ themes/ShinyMetal/focuslist/focuslist.cfg	2000-12-28 19:16:31.000000000 +0100
@@ -2,12 +2,12 @@
   __NAME "WARPFOCUS"
   __JUSTIFICATION 0
 
-  __NORMAL "zirkle/9"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_OUTLINE
   __FORGROUND_COLOR 255 180 0
   __BACKGROUND_COLOR 120 50 0
 
-  __CLICKED "zirkle/9"
+  __CLICKED "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_OUTLINE
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/ShinyMetal.orig/iconbox/iconbox.cfg themes/ShinyMetal/iconbox/iconbox.cfg
--- themes/ShinyMetal.orig/iconbox/iconbox.cfg	1999-10-19 05:54:38.000000000 +0200
+++ themes/ShinyMetal/iconbox/iconbox.cfg	2000-12-28 19:16:42.000000000 +0100
@@ -106,7 +106,7 @@
 
 __TCLASS __BGN
   __NAME "ICONBOX_WINDOW_TITLE"
-  __NORMAL "zirkle/9"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_OUTLINE
   __FORGROUND_COLOR 255 180 0
   __BACKGROUND_COLOR 120 50 0
diff -Nur themes/ShinyMetal.orig/init/init.cfg themes/ShinyMetal/init/init.cfg
--- themes/ShinyMetal.orig/init/init.cfg	1999-10-19 05:24:30.000000000 +0200
+++ themes/ShinyMetal/init/init.cfg	2000-12-28 19:16:55.000000000 +0100
@@ -1,5 +1,15 @@
 /* Shiny Metal 2 */
 
+/*
+Jan 27 2000, Japanese Localization by Masahiko Mori <masa@engr.sgi.com>
+                      Traslation by Ryuichi Wakita <wakita@xtrans.com>
+
+このファイルの日本語訳の部分、その他の変更部分は新規に作成したので上記２人に
+著作権があり、GPL に従って自由に配布できます。それ以外の変更のない部分につい
+ては原作の制限に従うものとします。
+このファイルに変更を加えた場合、このコメントに続き簡単な履歴を追加してください。
+*/
+
 BEGIN_BACKGROUND("STARTUP_BACKGROUND")
 	SET_SOLID("100 70 40")
 	ADD_BACKGROUND_SCALED("init/images/startup_background.jpg")
@@ -37,7 +47,7 @@
 
 __TCLASS __BGN
   __NAME "PROGRESS_TEXT_NUMBER"
-  __NORMAL "zirkle/9"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_OUTLINE
   __FORGROUND_COLOR 255 180 0
   __BACKGROUND_COLOR 120 50 0
diff -Nur themes/ShinyMetal.orig/pager/pager.cfg themes/ShinyMetal/pager/pager.cfg
--- themes/ShinyMetal.orig/pager/pager.cfg	1999-10-19 05:24:31.000000000 +0200
+++ themes/ShinyMetal/pager/pager.cfg	2000-12-28 19:17:06.000000000 +0100
@@ -1,3 +1,13 @@
+/*
+Jan 27 2000, Japanese Localization by Masahiko Mori <masa@engr.sgi.com>
+                      Traslation by Ryuichi Wakita <wakita@xtrans.com>
+
+このファイルの日本語訳の部分、その他の変更部分は新規に作成したので上記２人に
+著作権があり、GPL に従って自由に配布できます。それ以外の変更のない部分につい
+ては原作の制限に従うものとします。
+このファイルに変更を加えた場合、このコメントに続き簡単な履歴を追加してください。
+*/
+
 __ICLASS __BGN
   __NAME PAGER_SEL
   __NORMAL         "pager/images/p_sel.png"
@@ -33,7 +43,7 @@
 
 __TCLASS __BGN
   __NAME "PAGER_WINDOW_TITLE"
-  __NORMAL "zirkle/9"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_OUTLINE
   __FORGROUND_COLOR 255 180 0
   __BACKGROUND_COLOR 120 50 0 
diff -Nur themes/ShinyMetal.orig/textclasses.cfg themes/ShinyMetal/textclasses.cfg
--- themes/ShinyMetal.orig/textclasses.cfg	1999-10-19 05:54:42.000000000 +0200
+++ themes/ShinyMetal/textclasses.cfg	2000-12-28 19:17:49.000000000 +0100
@@ -1,3 +1,13 @@
+/*
+Jan 27 2000, Japanese Localization by Masahiko Mori <masa@engr.sgi.com>
+                      Traslation by Ryuichi Wakita <wakita@xtrans.com>
+
+このファイルの日本語訳の部分、その他の変更部分は新規に作成したので上記２人に
+著作権があり、GPL に従って自由に配布できます。それ以外の変更のない部分につい
+ては原作の制限に従うものとします。
+このファイルに変更を加えた場合、このコメントに続き簡単な履歴を追加してください。
+*/
+
 #include <definitions>
 
 __E_CFG_VERSION 0
@@ -10,7 +20,7 @@
 
 __TCLASS __BGN
   __NAME "TEXT1"
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -19,13 +29,13 @@
 
 __TCLASS __BGN
   __NAME "TEXT2"
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_NONE
   __FORGROUND_COLOR 0 0 0
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 1024
 
-  __NORMAL_ACTIVE "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL_ACTIVE "-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/ShinyMetal.orig/tooltips/tooltips.cfg themes/ShinyMetal/tooltips/tooltips.cfg
--- themes/ShinyMetal.orig/tooltips/tooltips.cfg	1999-10-19 05:24:31.000000000 +0200
+++ themes/ShinyMetal/tooltips/tooltips.cfg	2000-12-28 19:17:28.000000000 +0100
@@ -1,3 +1,13 @@
+/*
+Jan 27 2000, Japanese Localization by Masahiko Mori <masa@engr.sgi.com>
+                      Traslation by Ryuichi Wakita <wakita@xtrans.com>
+
+このファイルの日本語訳の部分、その他の変更部分は新規に作成したので上記２人に
+著作権があり、GPL に従って自由に配布できます。それ以外の変更のない部分につい
+ては原作の制限に従うものとします。
+このファイルに変更を加えた場合、このコメントに続き簡単な履歴を追加してください。
+*/
+
 __ICLASS __BGN
   __NAME DEFAULT_TOOLTIP_MAIN
   __NORMAL "tooltips/images/ttip_cloud_big.png"
@@ -32,7 +42,7 @@
 
 __TCLASS __BGN
   __NAME "TOOLTIP_TEXT"
-  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
+  __NORMAL "-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
