--- e16-theme-ShinyMetal-0.16.8/ShinyMetal/fonts.cfg~	2005-07-16 08:36:28.000000000 +0000
+++ e16-theme-ShinyMetal-0.16.8/ShinyMetal/fonts.cfg	2006-07-19 01:21:40.000000000 +0000
@@ -15,15 +15,15 @@
 font-epplet-medium	"-*-lucida-medium-r-normal-*-*-100-*-*-p-*-*-*,-*-helvetica-medium-r-normal-*-10-*"
 font-epplet-large	"-*-lucida-medium-r-normal-*-*-120-*-*-p-*-*-*,-*-helvetica-medium-r-normal-*-12-*"
 
-font-focus	"zirkle/9"
+font-focus	"-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*--14-*"
 
-font-iconbox	"zirkle/9"
+font-iconbox	"-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*--14-*"
 
-font-init	"zirkle/9"
+font-init	"-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*--14-*"
 
 font-menu	"-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
 
-font-pager	"zirkle/9"
+font-pager	"-*-helvetica-medium-r-*-*-12-*-*-*-*-*-*-*,-*-fixed-medium-r-normal--14-*,-*--14-*"
 
 font-tooltip	"-*-helvetica-medium-r-*-*-10-*-*-*-*-*-*-*"
 
